src/Bicep.LangServer/Options/BicepLangServerOptions.cs

```diff
@@ -0,0 +1,12 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Bicep.LanguageServer.Options;
+
+public class BicepLangServerOptions : IBicepLangServerOptions
```

Comment:
Could we simplify to make this a record and avoid creating an interface (unless the interface is necessary for testing)?

e.g.
```csharp
public record BicepLangServerOptions(
    bool VsCompatibilityMode);
```

Reply:
7a833789b80fae27020f7397d59a88f7cbf68618